ui/src/app/pages/SchemaMapping/MatchingSchemaCard.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2020 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import {
+  Grid,
+  GridItem,
+  Button,
+  ButtonVariant,
+  DescriptionListDescription,
+} from '@patternfly/react-core'
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon'
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon'
+
+export type MatchingSchemaProps = {
+  artifactName: string
+  keySchema: boolean
+}
+
+export const MatchingSchemaCard: React.FC<MatchingSchemaProps> = ({
+  artifactName,
+  keySchema,
```

Comment:
what is keySchema?

Reply:
The matching schema card will display the data i the format topic-name-key or topic-name-value.
keySchema determines whether description is for keySchema or valueSchema so as to display appropriate data